libs/deno/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "mt_deno"
+edition.workspace = true
```

Comment:
to change in all projects?

Reply:
I did start down that road but reverted after thinking it might be better to do it in another PR.